x-pack/plugins/serverless_search/public/application/components/languages/java.ts

```diff
@@ -14,14 +14,14 @@ export const javaDefinition: LanguageDefinition = {
   iconType: 'java.svg',
   github: {
     label: i18n.translate('xpack.serverlessSearch.languages.java.githubLabel', {
-      defaultMessage: 'elasticsearch-java-serverless',
+      defaultMessage: 'elasticsearch-java',
     }),
-    link: 'https://github.com/elastic/elasticsearch-java/tree/main/java-client-serverless',
+    link: 'https://github.com/elastic/elasticsearch-java/tree/main/java-client',
```

Comment:
The GitHub URL is `https://github.com/elastic/elasticsearch-java/` (we previously had a deep link to a specific directory).

Now I really think the URL should be `https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/current/` - our users are more interested in the documentation than the source code repository. However there is a GitHub icon before the URL. @joemcelroy, can we remove this icon?

Reply:
Ok. So let's point to the repository itself and not a specific branch.

```suggestion
    link: 'https://github.com/elastic/elasticsearch-java',
```